pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.401</version>
+        <version>2.21</version>
     </parent>
 
     <artifactId>versioncolumn</artifactId>
```

Comment:
@oleg-nenashev what do you have in mind? I was about to use like "Jenkins Versions Node Monitors Plugin", but then I'm wondering if as this plugin is very old this might not confuse people.
Well at the same time, long time Jenkins will understand at some point, and it will clarify possibly for newcomers.